f5_heat/resources/f5_sys_iappcompositetemplate.py

```diff
@@ -125,13 +92,13 @@ def _add_optional_attr(self, iapp_dict):
         :returns: possibly modified dictionary
         '''
 
-        if self.properties[self.COMPOSITE_TEMPLATE][self.REQUIRES_MODULES]:
+        if self.properties[self.REQUIRES_MODULES]:
             iapp_dict['requiresModules'] = \
```

Comment:
Is it possible to specify `self.properties` such that this conditional can be replaced with:  `iapp_dict.update(self.properties)` ?